front/src/applications/editor/components/EntityError.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import cx from 'classnames';
+import { BsFillExclamationOctagonFill } from 'react-icons/bs';
+import { uniqueId } from 'lodash';
+
+import { EditorEntity } from 'types';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { getInfraID } from 'reducers/osrdconf/selectors';
+import { InfraErrorLine } from './InfraErrors/InfraError';
+
+const EntityError: FC<{ entity: EditorEntity; className?: string }> = ({ entity, className }) => {
+  const { t } = useTranslation();
+  const infraID = useSelector(getInfraID);
+  const { data } = osrdEditoastApi.endpoints.getInfraByIdErrors.useQuery({
+    id: infraID || 0,
+    objectId: entity.properties.id,
+  });
+
+  const hasError = data?.results && data.results.length > 0;
+  return hasError ? (
```

Comment:
Only a suggestion to remove the data?.results? l29
```suggestion
  const errors = data?.results;
  return errors && errors.legnth > 0 ? (
```

and then
```
errors.map(...)
```